src/Sylius/Bundle/WebBundle/Controller/Backend/SecurityController.php

```diff
@@ -27,7 +27,7 @@
 class SecurityController extends Controller
 {
     /**
-     * Target action for _switch_user=_exit, redirects admin back to impersonated user
+     * Target action for _switch_user=_exit, redirects admin back to impersonated user.
```

Comment:
Good, but for the future, let's avoid non-related changes. It is better to open a separate PR with bunch of CS fixes. Much easier to review. :)